bigquery_etl/metadata/parse_metadata.py

```diff
@@ -59,7 +59,8 @@ class PartitionMetadata:
 
     field: str
     type: PartitionType
-    require_partition_filter: bool = attr.ib(False)
+    require_partition_filter: bool = attr.ib(True)
+    expiration_ms: Optional[int] = attr.ib(None)
```

Comment:
This is already merged, but additional wrinkle is that CREATE TABLE DDL uses days rather than ms, which is fascinating: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#table_option_list

That takes a float argument, so you can specify partial days, and I assume it ends up turning that into ms behind the scenes.

So, there's an argument for having this be days as a float, and handle the math to turn into millis.

Reply:
Yes, agreed. We should use days here instead.